comment/api/permissions.py

```diff
@@ -11,8 +11,11 @@
 class IsOwnerOrReadOnly(permissions.BasePermission):
     """
     Custom permission to only allow owner of an object to edit it.
+    Anonymous objects can't be edited.
     """
     def has_object_permission(self, request, view, obj):
+        if not obj.user:    # anonymous comment
```

Comment:
This statement is needless here because `obj.user == request.user` is always false for anonymous comment.

Reply:
You are right. I had just added this so that this gets executed before any check and the operation happens a bit faster maybe.

Reply:
Since we have no issue with the performance, it would be much better to keep the code cleaner.

Reply:
okay, seems legit. will remove it.